File: code/java/lecture2/Area.java
public class Area {
    public static void main(String[] args) {
        int radius = (int)(Math.random() * (9 - 0 + 1) + 0);
        double area =  Math.PI * (radius * radius);;
        System.out.println("The area of a circle with a radius of " + radius + " is " + area);
    }
}

File: code/ocaml/practice/area.ml
type figure =
  Circle of float                (* Circle(r) represents a circle whose radius is r *)
| Rectangle of float * float     (* Rectangle(w, h) represents a rectangle with width w and height h *)

let area figure =
  match figure with
    Circle(r) -> r *. r *. 3.14159
    | Rectangle(w, h) -> w *. h;;

area (Circle(2.0));;
area (Rectangle(3.0, 4.0));;